testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -458,13 +469,20 @@ def _check_atomgroup(self, ag, u):
         assert_equal(len(u.atoms[4].bonds), 0)
         assert_equal(len(u.atoms[5].bonds), 0)
 
-    def test_atomgroup_guess_bonds(self):
+    @pytest.mark.parametrize(
+        'ff, lb, nbonds',
+        [
+            (0.55, 0.1, 2), (0.9, 1.6, 1),
+            (0.5, 0.2, 2), (0.1, 0.1, 0)
+        ]
+    )
+    def test_atomgroup_guess_bonds(self, ff, lb, nbonds):
         """Test an atomgroup doing guess bonds"""
         u = mda.Universe(two_water_gro)
 
         ag = u.atoms[:3]
-        ag.guess_bonds()
-        self._check_atomgroup(ag, u)
```

Comment:
Why is this being removed?

Reply:
As it raises an assertion error each time a different bond number than the default value(2 bonds) was guessed.

Reply:
we can solve this by passing these parameters to _check_atomgroup, but would it be useful then?

Reply:
Let's leave the check_universe in, but only call it if the default parameters are used.

Reply:
Sure @ianmkenney ! It's actually a good idea.